maintainers/maintainer-list.nix

```diff
@@ -2122,6 +2122,7 @@
     email = "bdd@mindcast.org";
     github = "bdd";
     githubId = 11135;
+    matrix = "@bdd:mozilla.org";
```

Comment:
Put this in a separated commit

Reply:
Oof. This wasn't supposed to be part of this commit.
Amended.